docs/project/release-policy.md

```diff
@@ -0,0 +1,43 @@
+# Release Policy
+
+Crystal releases have a version indicated by a major, minor and patch number.
+
+The current major branch number is `1`.
+
+New features are added in minor releases (`1.x.0`) which are regularly scheduled every three months.
+
+Patch releases contain only important bug fixes and are released when necessary.
+They usually only appear for the latest minor branch.
+
+New releases are announced at [crystal-lang.org/releases](https://crystal-lang.org/releases) ([RSS feed](https://crystal-lang.org/releases)).
+
+There are currently no plans for a new major release.
+
+## Backwards compatibility
+
+Minor and patch releases are backwards compatible: Well-defined behaviours and documented APIs in a given version
+will continue working on future versions within the same major branch.
```

Comment:
```suggestion
Minor and patch releases are backwards compatible: Well-defined behaviours and documented APIs in a given version
will continue working on future versions within the same major branch.
This applies to all features listed in the [API docs](https://crystal-lang.org/api) and [language reference](https://crystal-lang.org/reference/syntax_and_semantics/). It covers any observable behaviour (unless its obviously invalid, i.e. a bug), even if the documentation does not explicitly mention it (i.e. the documentation is incomplete).
```

Reply:
I prefer mine, but ok :-D (behavior if you prefer US english)